content/configuring_tests/_index.md

```diff
@@ -25,6 +25,7 @@ Sakuli tests can be fine-tuned via various properties.
 | `sakuli.log.folder` | String: `${project.rootDir}`/_logs | Folder for log files  |
 | `sakuli.screenshot.onError` | Boolean: true | Enable / disable screenshots on error |
 | `sakuli.screenshot.dir` | String: `${project.rootDir}`/_logs/_screenshots | Folder for screenshot files (if activated) |
+| `sakuli.screenshot.storage` | String: `hierarchical` | Configures the way Sakuli stores the error screenshots |
```

Comment:
Please add all possible options for this config property. An explanation of every possible value would also be helpful.